### PR TITLE
Style user signout form

### PR DESCRIPTION
**Pull request by UAMS16**

### CODE REVIEW

File: cegs_portal/templates/account/logout.html [hunk 1]
```diff
@@ -1,19 +1,73 @@
...
+        background-color: #c02626;
+    }
+
+    h1 {
+        font-size: 1.25rem;
+        font-weight: bold;
+        line-height: 1.75rem;
+    }
+
+    .faded-line {
```
Comment (tncowart) [inline cegs_portal/templates/account/logout.html:51]: This class isn't used, I think you should remove it.

**Review by tncowart — CHANGES_REQUESTED** [review]
The "no" button doesn't look great:
Screenshot 2023-04-18 at 11 10 23 AM

And there's an unused class, but otherwise looks good.

File: cegs_portal/templates/account/logout.html (hunk 1 above)
Comment (UAMS16) [reply cegs_portal/templates/account/logout.html:51]: Thanks for the compliment, Thomas. Fixed the "No" button and removed the unused class.

Comment (UAMS16) [reply cegs_portal/templates/account/logout.html:51]: image

**Review by tncowart — APPROVED** [review]
Looks good!

**Pull request merged**